baybe/kernels/kernels.py

```diff
@@ -0,0 +1,18 @@
+"""Collection of usable kernels."""
+
+from attrs import define, field
+from attrs.validators import in_
+
+from baybe.kernels.base import Kernel
+
+
+@define
+class MaternKernel(Kernel):
+    """A Matern kernel using a smoothness parameter nu."""
+
+    nu: float = field(converter=float, validator=in_([0.5, 1.5, 2.5]), default=2.5)
+    """A smoothness parameter.
+
+    It only takes the values 0.5, 1.5 or 2.5. Smaller values are less smooth.
+    The value 2.5 is chosen as default.
```

Comment:
Avoid repeating defaults (refactorability!). The default will be automatically displayed in the doc

Reply:
Changed.